spec/controllers/admin/activities_controller_spec.rb

```diff
@@ -22,7 +22,7 @@
       it_behaves_like "unauthorized"
     end
 
-    %w[board communications translation tag_wrangling docs support open_doors].each do |role|
+    %w[board communications elections translation tag_wrangling docs support open_doors].each do |role|
```

Comment:
Maybe we should make things easier for our future selves and switch to an approach like `(Admin::VALID_ROLES - %w[superadmin policy_and_abuse support]).each do |role|` anywhere we're testing all of the roles except for a few.